code/game/gamemodes/cult/runes.dm

```diff
@@ -141,9 +142,11 @@ structure_check() searches for nearby cultist structures required for the invoca
 		animate(src, transform = matrix()*2, alpha = 0, time = 5) //fade out
 		animate(transform = oldtransform, alpha = 255, time = 0)
 
-/obj/effect/rune/proc/fail_invoke(var/mob/living/user)
+/obj/effect/rune/proc/fail_invoke(message=TRUE)
 	//This proc contains the effects of a rune if it is not invoked correctly, through either invalid wording or not enough cultists. By default, it's just a basic fizzle.
-	visible_message("<span class='warning'>The markings pulse with a small flash of red light, then fall dark.</span>")
+	if(message)
+		visible_message("<span class='warning'>The markings pulse with a \
+			small flash of red light, then fall dark.</span>")
 	spawn(0) //animate is a delay, we want to avoid being delayed
```

Comment:
Also I'd like to note I really liked the 'cultists trying to invoke it made it flash' thing and would be sad if it just did so automatically.
Maybe, on line 474, check if there ARE any invokers before using fail_invoke()?